terraform/mysql/provider.tf

```diff
@@ -0,0 +1,19 @@
+terraform {
+  required_providers {
+    aiven = {
+      source = "aiven/aiven"
+    version = ">= 2.0.0, < 3.0.0" }
```

Comment:
the latest version of the Aiven terraform provider is 3.10.0. I see this comes from the docs though, so perhaps they are out of date

Reply:
Made the changes